content/en/docs/collector/troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Troubleshooting
+weight: 32
+---
+
+The OpenTelemetry Collector supports a variety of metrics, logs, and extensions
+for troubleshooting collector health and performance.
```

Comment:
The opening paragraph should let the reader know what they'll find in this page.